content/docs/tasks/security/authn-policy/index.md

```diff
@@ -340,4 +340,5 @@ $ kubectl delete ns foo bar legacy
 
 ## What's next
 
-* Learn more about verifying mutual TLS setup [testing Istio mutual TLS authentication](/docs/tasks/security/mutual-tls/)
+* Learn more about verifying mutual TLS setup [testing Istio mutual TLS authentication](/docs/tasks/security/mutual-tls/).
+* Learn about how to [enable mutual TLS incrementally](https://istio.io/docs/tasks/telemetry/using-istio-dashboard/)).
```

Comment:
Should we mention a little bit about the permissive mode in this page? This is an important feature provided by authPolicy, you don't have to provide example, but just something like "Besides plain text and mutual TLS, AuthPolicy provides a permissive mode that can accept both traffic, which is important for service migration, refer migration task for more info". And we may put it something in the middle of this page, not this section.

Reply:
Done.